templates/python/.github/workflows/publish.yaml

```
@@ -0,0 +1,29 @@
+name: Publish
+# Template adapted from https://github.com/bakdata/ci-templates/tree/main/.github/workflows#python-poetry-release
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  release:
+    name: release
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@main
+    with:
+      release-type: patch publish-to-test: true
+      python-version: 3.8
```

this should also match what's specified in `pyproject.toml`
```suggestion
      python-version: "3.10"
```

the quotes could be important for 3.10

---

done